submissions/Dovahkiin1991/html-css-popup/index.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hooli-style Popup</title>
+    <link rel="stylesheet" href="./css/styles.css">
+</head>
+<body>
+    <header class="header header--sticky">
+        <nav>
+            <ul class="navbar">
+                <li class="navbar__item">
+                    <input type="checkbox" id="navbar1">
+                    <label class="navbar__link" for="navbar1">
+                        <img src="./images/icons/popup-button.png" alt="Popup Icon">
+                    </label>
+                    <ul class="navbar navbar__sub navbar__sub--icons">
```

Comment:
Please fix the popup size, it's hard to understand what content is inside the popup
![image](https://user-images.githubusercontent.com/16196199/187022734-ed4966ae-d1c2-42af-b784-eae43252d4c0.png)